docs/functions.md

```diff
@@ -205,7 +205,7 @@ The **Register** action registers a function as an event handler. The function i
 ```python
 class GearsBuilder.register(regex='*', mode='async', batch=1, duration=0,
   eventTypes=None, keyTypes=None, onRegistered=None, onFailedPolicy="continue",
-  onFailedRetryInterval=1)
+  onFailedRetryInterval=1, trigger=None)
```

Comment:
@itamarhaber I am not sure about this session anymore, basically, the register gets a kargs now and each reader extracts what it needs.

Reply:
I see.

So, are there any args that apply to all readers here?

Reply:
Only the `onRegistered=None`, `mode`, `regex (which was changed to prefix)`
I will fix it

Reply:
Know what, its actually just the `onRegistered=None` because of command reader